Packs/PAN-OS/.secrets-ignore

```diff
@@ -13,4 +13,5 @@ https://test_pb_task.not.real
 10.10.0.12
 10.10.0.0
 10.10.0.1
-10.10.0.1
\ No newline at end of file
+10.10.0.1
+capCo
```

Comment:
It is not a good practice to add this name here. Please change the name of your tests class instead.

Reply:
done, why does it recognize it as a secret anyway?